### PR TITLE
update class and subclass

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

File: modules/ROOT/pages/errors/all-errors.adoc [hunk 1]
```diff
@@ -4,16 +4,16 @@
 [[neo4j-errors]]
 = List of all error codes
 
-This page contains lists of all Neo4j errors, grouped by class.
+This page contains lists of all Neo4j errors, grouped by type.
 
 == Client errors
 
-This is a complete list of all client error status codes Neo4j may return, and what they mean.
+This is a complete list of all client error codes Neo4j may return, and what they mean.
```
Comment (Lojjs) [inline modules/ROOT/pages/errors/all-errors.adoc:11]: I think it is OK if we like to use a separate wording than `Neo4j status code` for errors. But now there is a mix of e.g. `[client] error codes`, `[transient] error status code`, `Neo4j error code`. I think we should try to be more consistent. 

Also if we not decide to call it just `Neo4j status codes`, I think we should add a note to clarify that it is a subset of `Neo4j status codes`

File: modules/ROOT/pages/notifications/index.adoc [hunk 2]
```diff
@@ -57,11 +57,11 @@ The following categories exist, ordered by severity:
 | SECURITY
 | WARNING OR INFORMATION
 | The result of the query or command indicates a potential security issue.
-| Make sure that the behaviour is what you intended.
+| Make sure that the behavior is what you intended.
 
 | TOPOLOGY
 | INFORMATION
-| Information provided while executing database and server related commands.
+| Information provided while executing database and server-related commands.
```
Comment (Lojjs) [inline modules/ROOT/pages/notifications/index.adoc:64]: Do you think we should update the GENERIC one to also not say 'wider class' but 'another category' or something similar, or do you think it is ok to use the word 'class' there? 

Also a side note, drivers team is at least partly using these descriptions in their configuration API, not sure if they like to adapt to language changes we do here.

**Review by Lojjs — COMMENTED** [review]
I think the type/subtype thing looks correct, but had two comments on the related changes.

File: modules/ROOT/pages/notifications/index.adoc (hunk 2 above)
Comment (renetapopova) [reply modules/ROOT/pages/notifications/index.adoc:64]: I wasn't sure if this was something from the code. I can remove the dash.

**Review by Lojjs — COMMENTED** [review]
Some suggestions to make it even more aligned and clear how the different terms are related to each other

**Pull request merged**